packages/gestalt/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gestalt",
-  "version": "0.77.0",
+  "version": "0.77.1",
```

Comment:
I don't think we want to bump up the version until we are doing a release. this should be kept the same.